woocommerce/woocommerce-ios
# Product creation AI: Present sheet to select AI tone

Closes: #10749 Closes: #10755

## Description

Adds a way to select AI tone and voice from the features screen.

- Presents a bottom sheet to select AI tones.
- Stores the selected tone in user defaults.
- Clear the value upon logout. 

## Testing instructions
- Create a JN site with the Jetpack AI plugin. (Checkbox available on JN home page to install this plugin)
- Login into the app using the JN site
- Switch to the Products tab and select the "+" button to add a new product. 
- Tap "Create a product with AI" option
- Enter a product name and tap "Continue"
- Tap on "Set tone and voice" and validate that the AI tone selection sheet is launched
- Select a tone and dismiss the sheet
- Reopen the sheet and ensure that the selection is restored

## Screenshots

https://github.com/woocommerce/woocommerce-ios/assets/524475/4e7d3ea3-069d-4796-8a90-8a74be9ed685


---
